orttraining/orttraining/core/framework/module_gradient_graph_builder.h

```diff
@@ -50,6 +50,7 @@ struct TrainingGraphInfo {
   // Indices of output grads that need to be materialized to full size all-0 tensor.
   // Otherwise, we can use scalar-0 tensor.
   std::vector<size_t> output_grad_indices_require_full_shape{};
+  std::vector<std::string> ort_break_op_output_names{};
```

Comment:
>ort_break_op_output_names [](start = 27, length = 25)

this name is exposed in ortmodule frontend, and the user doesn't necessarily  understand what's a BreakOp. 

I think a better name would be "output_grad_names" .
and in the comment you can explain this would be the output of BreakOp.